patching/README.md

```diff
@@ -0,0 +1,97 @@
+# Patching Sample
+
+This sample shows how to safely alter a workflow using `patched` and `deprecate_patch` in stages.
```

Comment:
Why did you choose this workflow as the sample for patches?
It's not especially practical and doesn't show how patches help making workflow changes that would otherwise break determinism, changing a query result can be done without patching.

Reply:
I didn't really choose one, I just wanted to focus on the patch part without creating activities or timers.

Reply:
I have updated the sample to use activities instead. I was gonna demonstrate the non-determinism in README, but I can't until I upgrade Python to a core version with https://github.com/temporalio/sdk-core/pull/475.